### ISSUE TITLE
Unexpected token ?

### ISSUE THREAD

**Issue by r-bt** [noahsalvi/sapper-netlify#1]
Not sure why this is happening but when I run `npx sapper-netlify .` I just get 

```
npx: installed 1 in 4.204s
Unexpected token ?
```

**Comment by noahsalvi** [comment]
Weird what happens when you run `npx sapper-netlify@latest .`

**Comment by r-bt** [comment]
Hum...same thing

**Comment by noahsalvi** [comment]
Are you using windows?

**Comment by r-bt** [comment]
No, I'm using macOS. It's grand I'll clone the repo and see can I debug what's going on

**Comment by r-bt** [comment]
Got it, I'm using Node v12 (I've got another project with Firebase cloud functions which currently doesn't support v12) and the`Nullish Coalescing` operator for:

```javascript
function installSapper() {
  return exec(
    `npx degit "sveltejs/sapper-template#rollup" ${args[0]} ${args[1] ?? ""}`
  );
}
```

 is only available in v14. Updating to node v14 fixes this problem

**Comment by noahsalvi** [comment]
Ahh that makes sense I'll fix it thanks for finding it

**Comment by noahsalvi** [comment]
Fixed in `0.2.2`
